### PR TITLE
[Task] Fix Error in Console

### PR DESCRIPTION
**Pull request by Habiburr0hman**
## Story/Task
Fix some errors in the browser console

## Summary
- Fix errors in console, mostly because "each child in a list should have a unique 'key' prop"

## Test Plan
- Open some pages in website
- Check console whether there are some errors or not

## Screenshot
Screenshot (3851)

### CODE REVIEW

File: src/components/BlogPost.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ const BlogPost: React.FC<BlogPostCard> = (props) => {
         primary_author,
         excerpt: description,
         feature_image,
-        url: link,
+        // url: link,
```
Comment (IloveNooodles) [inline src/components/BlogPost.tsx:15]: knp di komen?

File: src/components/BlogPost.tsx [hunk 2]
```diff
@@ -59,14 +59,14 @@ const BlogPost: React.FC<BlogPostCard> = (props) => {
                             >
                                 {formattedDate} | Posted by {authorName}
                             </p>
-                            <a
-                                href={link}
```
Comment (IloveNooodles) [inline src/components/BlogPost.tsx:63]: ini jadi gabisa di klik dong

**Review by IloveNooodles — APPROVED** [review]
approved with some comment

File: src/components/BlogPost.tsx (hunk 2 above)
Comment (Habiburr0hman) [reply src/components/BlogPost.tsx:63]: Saya coba masih bisa kak, nggak tau juga kenapa wkwk. Soalnya kalau tetep tag a itu ada error nested DOM kak

File: src/components/BlogPost.tsx (hunk 1 above)
Comment (Habiburr0hman) [reply src/components/BlogPost.tsx:15]: Ini karena yang bawah tadi hrefnya dikomen, jadi variabelnya nggak terpakai kak

**Pull request merged**